index.html

```diff
@@ -906,7 +906,7 @@ <h2>Definition of Roles</h2>
 				<ol>
 					<li>Associate the content with a focusable element using <pref>aria-labelledby</pref> or <pref>aria-describedby</pref>.</li>
 					<li>Place the content in a focusable element that has role <rref>document</rref> or <rref>article</rref>.</li>
-					<li>Manage focus of descendants as described in <a href="#managingfocus">Managing Focus</a>, updating the value of <pref>aria-activedescendant</pref> to reference the <a>element</a> containing the focused content.</li>
+					<li>Manage focus of <a>owned</a> elements is described in <a href="#managingfocus">Managing Focus</a>, updating the value of <pref>aria-activedescendant</pref> to reference the <a>element</a> containing the focused content.</li>
```

Comment:
Change "is" to "as" (because this point is part of "authors MUST use one of the following techniques"):
```suggestion
					<li>Manage focus of <a>owned</a> elements as described in <a href="#managingfocus">Managing Focus</a>, updating the value of <pref>aria-activedescendant</pref> to reference the <a>element</a> containing the focused content.</li>
```